samples/msal-node-samples/auth-code-cli-app/src/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const { PublicClientApplication, InteractionRequiredAuthError } = require("@azure/msal-node");
+const open = require("open");
+const { authConfig } = require("./authConfig.js");
+const { callMicrosoftGraph } = require("./graph.js");
+
+// Before running the sample, you will need to replace the values in the config.
+
+
+// Get url to sign user in and consent to scopes needed for application
```

Comment:
Can we have a more descriptive comment here, and a remark about the **open** dependency?

Reply:
Done